extension/src/popup/views/UnlockBackupPhrase.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Formik } from "formik";
+
+import { showBackupPhrase } from "@shared/api/internal";
+
+import { POPUP_WIDTH } from "constants/dimensions";
+
+import { emitMetric } from "helpers/metrics";
+
+import { METRIC_NAMES } from "popup/constants/metricsNames";
+import { ROUTES } from "popup/constants/routes";
+import { history } from "popup/constants/history";
+import { COLOR_PALETTE, FONT_WEIGHT } from "popup/constants/styles";
+
+import { BackButton } from "popup/basics/Buttons";
+import {
+  Form,
+  SubmitButton,
+  FormRow,
+  ApiErrorMessage,
+  TextField,
+} from "popup/basics/Forms";
+
+const UnlockAccountEl = styled.div`
+  width: 100%;
+  max-width: ${POPUP_WIDTH}px;
+  box-sizing: border-box;
+  padding: 2.25rem 2.5rem;
+`;
+export const HeaderContainerEl = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 2.5rem 0.25rem;
+  line-height: 1;
+`;
+export const HeaderEl = styled.h1`
+  color: ${COLOR_PALETTE.primary}};
+  font-weight: ${FONT_WEIGHT.light};
+  margin: 1rem 0 0.75rem;
+`;
+const SubheaderEl = styled.h3`
+  font-weight: ${FONT_WEIGHT.bold};
+  font-size: 0.95rem;
+  letter-spacing: 0.1px;
+  color: ${COLOR_PALETTE.primary}};
+`;
+
+const CustomFormTextFieldEl = styled(TextField)`
+  padding-right: ${(props) => (props.error ? "6rem" : "2.2rem")};
+`;
+
+const ButtonRowEl = styled.div`
+  padding: 1.5rem 0;
+`;
+
+export const UnlockBackupPhrase = () => {
+  const destination = ROUTES.displayBackupPhrase;
+
+  const [errorMessage, setErrorMessage] = useState("");
+
+  interface FormValues {
+    password: string;
+  }
+  const initialValues: FormValues = {
+    password: "",
+  };
+
+  const handleSubmit = async (values: FormValues) => {
+    const { password } = values;
+    const res = await showBackupPhrase(password);
```

Comment:
I decided to call this internal api within a component itself since on `fulfilled`, we aren't changing any state. All we need is an error message from `popupMessageListener.ts`